internal/node/processmanager/process_mgr_windows.go

```diff
@@ -0,0 +1,28 @@
+//go:build windows
+
+package processmanager
+
+import (
+	"context"
+	"errors"
+	"log/slog"
+
+	"github.com/synadia-io/nex/internal/models"
+	"github.com/synadia-io/nex/internal/node/observability"
+)
+
+// Initialize an appropriate agent process manager instance based on the sandbox config value
+func NewProcessManager(
+	log *slog.Logger,
+	config *models.NodeConfiguration,
+	telemetry *observability.Telemetry,
+	ctx context.Context,
+) (ProcessManager, error) {
+	if config.NoSandbox {
```

Comment:
this is an if statement that should never return true. Sandbox _can't_ be enabled on windows or mac/darwin.